src/components/scenes/CryptoExchangeQuoteScene.tsx

```diff
@@ -97,6 +98,22 @@ export class CryptoExchangeQuoteScreenComponent extends React.Component<Props, S
     ))
   }
 
+  showExplanationForCanBePartial = async () => {
```

Comment:
This method name needs to start with `handle` to satisfy a lint rule we plan to enable one day. Maybe `handleCanBePartialExplanation`

Reply:
This was to be consistent with other handler `showExplanationForEstimate`

I'll fix with a cleanup to the other handler as well